docs/terminal/getting-started.md

```diff
@@ -0,0 +1,156 @@
+---
+Order: 1
+Area: terminal
+TOCTitle: Getting Started Tutorial
+ContentId: 7B4DC928-2414-4FC7-9C76-E4A13D6675FE
+PageTitle: Getting started with the integrated terminal
+DateApproved: 06/05/2024
+MetaDescription: Learn how to get started running shell commands with the integrated terminal in Visual Studio Code.
+---
+# Getting started with the terminal
+
+Visual Studio Code includes a fully featured integrated terminal. You can use it to run commands like `echo`, `ls`, and `git`, just like a standalone terminal. The terminal in VS Code also provides integration with the editor to support features such as links to workspace files and error detection.
+
+The integrated terminal can use various shells installed on your machine. A shell is a program that interprets and executes the commands by interacting with the operating system. Examples of shells include Bash, Zsh, and PowerShell.
```

Comment:
Should we refer to it only once in the intro as the integrated terminal and then switch to simply "terminal" for the rest of the page?